sass/base/_btn.scss

```diff
@@ -121,25 +114,13 @@ Style guide: buttons
       color: $white;
     }
 
-    &:active,
-    &:focus {
-      outline: 3px solid $yellow-light;
-      outline-offset: -3px;
-    }
-
-    &:focus {
-      border-color: $yellow-light;
-      outline-color: $yellow-light;
-    }
+    @include _focus_outline_border(3px, $yellow-light, -3px);
 
     &::after {
       @extend .icon--after;
+      @extend .icon--right;
       @extend .icon--arrow-down;
-      position: relative;
-      top: -2px;
-      margin-left: 6px !important;
-      font-size: 0.45em;
-      @include transition(all 0.3s ease 0s);
```

Comment:
This implementation to have the arrow animated!
I think several is being removed without having any relation with the ticket itself!